definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-sol-brexit-nonprod.json

```diff
@@ -0,0 +1,58 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
I know it doesn't matter, but please update the date. It helps a very little bit to investigate when you see the field was added in 2021 and not in 2017 ;)